style2.scss

```diff
@@ -0,0 +1,315 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  background-repeat: no-repeat;
+  box-sizing: inherit;
+}
+a {
+  text-decoration: none;
+}
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+// Navigation
+nav {
```

Comment:
This looks ok structurally but class-based selectors are always a better option. Including a single line of space will definitely Improve readability.